app/models/host/base.rb

```diff
@@ -195,19 +201,21 @@ def set_interfaces(parser)
         self.primary_interface.save!
       end
 
+      changed_count = 0
       parser.interfaces.each do |name, attributes|
         iface = get_interface_scope(name, attributes).try(:first) || interface_class(name).new(:managed => false)
         # create or update existing interface
-        set_interface(attributes, name, iface)
+        changed_count += 1 if set_interface(attributes, name, iface)
       end
 
       ipmi = parser.ipmi_interface
       if ipmi.present?
         existing = self.interfaces.where(:mac => ipmi[:macaddress], :type => Nic::BMC.name).first
         iface = existing || Nic::BMC.new(:managed => false)
         iface.provider ||= 'IPMI'
-        set_interface(ipmi, 'ipmi', iface)
+        changed_count += 1 if set_interface(ipmi, 'ipmi', iface)
       end
+      telemetry_increment_counter(:importer_facts_count_interfaces, changed_count, type: parser.class_name_humanized)
```

Comment:
This line is unexpected in some tests that use a mocked parser, could you check that out?

Reply:
Don't understand this one?